pkg/apis/pipeline/v1/pipelinerun_types.go

```diff
@@ -427,6 +427,9 @@ type PipelineRunStatusFields struct {
 	// FinallyStartTime is when all non-finally tasks have been completed and only finally tasks are being executed.
 	// +optional
 	FinallyStartTime *metav1.Time `json:"finallyStartTime,omitempty"`
+
+	// Provenance contains all the information that needs to be recorded in a provenance i.e. ConfigSource
```

Comment:
I'd improve the docs here - this is tautological (i.e. "a foo is a foo") and references what looks like another type that does exist (assume users don't know about SLSA).

Reply:
(also in the other locations where this comment is used)

Reply:
Great point. I just updated the document for this field and the type as well. Please take a look. Happy to update it further for better doc!
Thanks @wlynch